.gitignore

```diff
@@ -25,3 +25,8 @@ lib/bs
 .merlin
 .bsb.lock
 node_modules
+src/InitiatingAuth.re
```

Comment:
I suggest making more targeted use of `git add` and `git rm` for local work rather than adding this sort of thing to a global ignore file.  You do lose some shortcuts (`git add .`, `git commit -A ...`) but that pain is nothing compared to losing a bunch of work because of an overly-ambitious ignore file, trust me :cry:.

Reply:
Don't know how to do that, actually. Will have to read up on it. That is, git strategies for local work. I will keep the .git file clean by just not adding anything to it!

Reply:
I have only rebased once. Not sure how to rebase this pull onto you new api. Any suggestions?

Reply:
I guess I say rebase but I guess I don't mean you literally have to rebase.  You can just merge in my changes into this branch.  The "command line instructions" below should explain that.